dev/archery/archery/crossbow/reports.py

```diff
@@ -160,6 +160,29 @@ def url(self, query):
         repo_url = self.job.queue.remote_url.strip('.git')
         return '{}/branches/all?query={}'.format(repo_url, query)
 
+    def todayStr(self):
+        date = datetime.now()
```

Comment:
Suggestion, here use `datetime.utcnow()` instead of `datetime.now()` so it won't suffer of random timezones depending on the system where the code will be executed on.